hoomd/hpmc/ComputeFreeVolume.h

```diff
@@ -45,13 +45,27 @@ class ComputeFreeVolume : public Compute
         //! Destructor
         virtual ~ComputeFreeVolume() { };
 
+        //! Get the number of MC samples to perform
+        unsigned int getNumSamples()
+            {
+            return m_n_sample;
+            }
+
         //! Set the number of MC samples to perform
+        //! \param n_sample number of MC steps to sample
         void setNumSamples(unsigned int n_sample)
             {
             m_n_sample = n_sample;
             }
 
+        //! Get the type of depletant particle
+        unsigned int getTestParticleType()
```

Comment:
For this to work with the associated `param_dict` entry `getTestParticleType` needs to return the string name of the particle type and `setParticleType` needs to take in a string argument. Perform the needed string<->int mappings in C++.